app/src/androidTest/java/com/github/ybecker/epforuml/QuestionDetailsTest.kt

```diff
@@ -55,6 +65,92 @@ class QuestionDetailsTest {
         onView(withId(R.id.recycler_forum)).check(matches(isDisplayed()))
     }
 
+    // if text is empty, no new answer is posted
+
+
```

Comment:
Usually I would advise to skip all the return lines because it makes the code super messy plus tricks codeclimate.